packages/block-library/src/icon/block.json

```
@@ -0,0 +1,62 @@
+{
+	"apiVersion": 3,
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"name": "core/icon",
+	"title": "Icon",
+	"category": "media",
+	"description": "Insert an SVG icon or graphic.",
+	"keywords": [ "icon", "svg" ],
+	"attributes": {
+		"icon": {
+			"type": "string",
+			"source": "html",
+			"selector": ".icon-container",
+			"default": "",
+			"__experimentalRole": "content"
+		},
+		"iconName": {
+			"type": "string",
+			"__experimentalRole": "content"
+		},
```

```suggestion
		"icon": {
			"type": "string",
			"source": "html",
			"selector": ".icon-container",
			"default": "",
			"role": "content"
		},
		"iconName": {
			"type": "string",
			"role": "content"
		},
```

`__experimentalRole` has been stabilized